android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginViewModel.kt

```diff
@@ -59,13 +60,22 @@ constructor(
   val launchDialPad
     get() = _launchDialPad
 
+  // -> user-resp (failure)
+  //    -> show-error
+  // -> user-resp (success)
+  //    -> store user info
+  //    -> goto home
```

Comment:
This is a public property so we can have the above comment in a detailed and formatted comment paragraph